loop-fs-iso-impl/src/main/java/com/github/stephenc/javaisotools/loopfs/iso9660/Iso9660FileSystem.java

```diff
@@ -52,7 +52,7 @@ public InputStream getInputStream(Iso9660FileEntry entry) {
         return buf;
```

Comment:
getBytes needs to be fixed too; it can't properly return a byte array of all the bytes when the entry's size is > Integer.MAX_VALUE (in Java you can't make a byte array with greater than Integer.MAX_VALUE as the length). I would add this to the top of getBytes:

```
if (entry.getSize() > Integer.MAX_VALUE) {
    throw new IOException("Entry too large");
}
```